src/PKSim.Presentation/Presenters/PopulationAnalyses/PopulationAnalysisOutputSelectionPresenter.cs

```diff
@@ -90,23 +99,16 @@ public void AddOutput()
 
       private void addOutput(QuantitySelectionDTO output)
       {
-         addOutput(new[] {output});
+         //Do not add simulation name as the output are already constructed as the intersection of all outputs available for all simulations
+         var displayPathAsString = _quantityDisplayPathMapper.DisplayPathAsStringFor(output.Quantity, addSimulationName: false);
```

Comment:
This will be a unified path were organism will be removed and simulation will not be visible

Reply:
For individual simulations: can it be done the way that:
* in individual simulation **comparison** chart simulation name is added, but
* in individual simulation chart simulation name is not added

?